tcms/report/tests.py

```diff
@@ -45,7 +45,8 @@ def test_report_by_caserun_tester_loads(self):
                 run=run,
                 build=product_build)
 
-        self.untested_tcr = TestCaseRunFactory(
+        # Untested testcase runs
```

Comment:
This comment is bogus. I can see that this creates an object but I have no idea why it doesn't get assigned to any variable. If you are adding comments don't tell me WHAT the code does. Instead describe WHY we do it in this way.